.pre-commit-config.yaml

```diff
@@ -38,18 +38,6 @@ repos:
                "python/cudf_kafka/cudf_kafka",
                "python/dask_cudf/dask_cudf"]
         pass_filenames: false
-  - repo: https://github.com/PyCQA/pydocstyle
-    rev: 6.3.0
-    hooks:
-      - id: pydocstyle
-        # https://github.com/PyCQA/pydocstyle/issues/603
-        additional_dependencies: [tomli]
-        args: ["--config=pyproject.toml"]
-        exclude: |
-          (?x)^(
-            ^python/cudf/cudf/pandas/scripts/.*|
-            ^python/cudf/cudf_pandas_tests/.*
-          )
```

Comment:
*Suggestion*: should we update the ruff version we're using to current from:
```
    rev: v0.1.13
```

To
```
    rev: v0.3.3
```

Reply:
Sure thing. Bumped ruff too

Reply:
Looks like `v0.3.3` formats quite a few new things by default. I'll bump ruff in a separate PR